docs/get-started/store-and-retrieve/set-up.md

```diff
@@ -39,233 +39,7 @@ A lite-node lets your computer interact with the Filecoin network without having
 
 To install a Lotus lite-node on your computer, you must have the tools required to _build_ a Lotus binary from the GitHub repository.
 
-| [MacOS](#macos) | [Ubuntu](#ubuntu) |
```

Comment:
Deleted this section because there are already installation instructions for macos and ubuntu